r1cs-std/src/fields/fp/mod.rs

```diff
@@ -584,6 +584,14 @@ impl<F: PrimeField> Clone for FpGadget<F> {
 }
 
 impl<F: PrimeField> AllocGadget<F, F> for FpGadget<F> {
+    #[inline]
+    fn alloc_constant<T, CS: ConstraintSystem<F>>(mut cs: CS, t: T) -> Result<Self, SynthesisError>
+    where
+        T: Borrow<F>,
+    {
+        Self::zero(cs.ns(|| "zero"))?.add_constant(cs.ns(|| "add constant"), t.borrow())
```

Comment:
Underneath the hood it gets reduced to something like `LinearCombination::from((constant, CS::one())`, so it might clearer to directly say that

Reply:
True. I can change it to that.